### PR TITLE
Added CTE-630 Config and edited CTE-440 Config

### PR DESCRIPTION
**Pull request by danielyko1017**
Tested by me personally

Proof of troubleshooting for the configs: https://discord.com/channels/615607687467761684/787072060563128321/834878502141558884

### CODE REVIEW

File: OpenTabletDriver/Configurations/Wacom/CTE-440.json [hunk 1]
```diff
@@ -1,63 +1,107 @@
...
+      },
+      "VendorID": 1386,
+      "ProductID": 21,
+      "InputReportLength": 5,
+      "OutputReportLength": null,
+      "ReportParser": "OpenTabletDriver.Tablet.TabletReportParser",
+      "FeatureInitReport": "AgI=",
+      "OutputInitReport": null,
+      "DeviceStrings": {},
+      "InitializationStrings": []
```
Comment (Kuuuube) [inline OpenTabletDriver/Configurations/Wacom/CTE-440.json:90]: In your verification this endpoint is market as `"CanOpen": false,` so it won't ever be used. Remove this.

File: OpenTabletDriver/Configurations/Wacom/CTE-440.json [hunk 2]
```diff
@@ -1,63 +1,107 @@
...
+        "Start": 0,
+        "StartInclusive": false,
+        "End": null,
+        "EndInclusive": false
+      },
+      "VendorID": 1386,
+      "ProductID": 21,
+      "InputReportLength": 10,
+      "OutputReportLength": null,
+      "ReportParser": "OpenTabletDriver.Tablet.TabletReportParser",
```
Comment (Kuuuube) [inline OpenTabletDriver/Configurations/Wacom/CTE-440.json:42]: Why did you change this parser. Your tablet doesn't use this endpoint and there is no verification that it needs to be changed.

**Review by Kuuuube — COMMENTED** [review]
Please fix your json formatting in this.

**Review by Kuuuube — APPROVED** [review]

File: OpenTabletDriver/Configurations/Wacom/CTE-440.json [hunk 3]
```diff
@@ -1,63 +1,107 @@
...
-      "InitializationStrings": []
-    }
-  ],
-  "Attributes": {}
+{
+  "Name": "Wacom CTE-440",
+  "DigitizerIdentifiers": [
+    {
+      "Width": 127.6,
+      "Height": 92.8,
```
Comment (gonX) [inline OpenTabletDriver/Configurations/Wacom/CTE-440.json]: You're changing `width` and `height`, and I'm now wondering if `MaxX` or `MaxY` also needs to be changed? The dimensions were changed away from 127.6cm/92.8cm in ce1c1270 to bring them in line with their reported LPI (calculated from MaxX/MaxY)

If `MaxX` and `MaxY` is correct, can you try to measure as accurately as possible whether this is the actual tablet area size? Also @Kuuuube FYI, since you made those corrections initially.

File: OpenTabletDriver/Configurations/Wacom/CTE-630.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{
+  "Name": "Wacom CTE-630",
+  "DigitizerIdentifiers": [
+    {
+      "Width": 208.8,
+      "Height": 150.8,
```
Comment (gonX) [inline OpenTabletDriver/Configurations/Wacom/CTE-630.json:6]: Like the other comment I left, please confirm that both the dimensions and `MaxX`/`MaxY` are accurate.

**Comment by jamesbt365** [comment]
superceded by #1658
